acceptance/scripts/generic/testrun.sh

```diff
@@ -2,9 +2,8 @@
 set -x
 
 export GEM_SOURCE=http://rubygems.delivery.puppetlabs.net
-
-export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-redhat6-64ma-debian6-64a-windows2008r2-64a}"
-export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-acceptance/suites/tests/00_smoke}"
+export GENCONFIG_LAYOUT="${GENCONFIG_LAYOUT:-redhat6-64ma-ubuntu1404-64a-windows2008r2-64a}"
+export BEAKER_TESTSUITE="${BEAKER_TESTSUITE:-acceptance/suites/tests/}"
```

Comment:
I have no idea.  I don't remember making this change.
Ultimately, it doesn't really matter because CI controls what OSes get tested.